pkg/resources/db/resource_db.go

```diff
@@ -1,4 +1,4 @@
-package resources
```

Comment:
Could it be better to call it resourcedb ? so that we are prepared in case we need a datasource of some of them, we can make the difference between resources and datasource packages

Reply:
done